### PR TITLE
Add link to documentation from readme

### PR DESCRIPTION
**Pull request by allhailwesttexas**
Had to go digging a little bit to find the actual documentation beyond the readme. Include a link in the readme to make it easier to find.

### CODE REVIEW

**Comment by agronholm** [comment]
The documentation link is already at the very top:
image
In addition it's linked directly from the PyPI: 
image

**Comment by allhailwesttexas** [comment]
The badge link is there, but I didn't spot it because I expected those types of labels to generally lead to a CI job of some sort, like the output of a job that builds the documentation. To find the docs I went to the PyPI page. It seemed reasonable to me to have a link to the full documentation in the repo readme, so I looked at one of your other repos (AnyIO) to see what was done there, and it has the docs badge + a section after the summary that explicitly links to the docs. I then made a similar change here. If you're happy with the link from the docs badge, feel free to just close this PR, but I still think it's an improvement.

File: README.rst [hunk 1]
```diff
@@ -34,6 +34,8 @@ their services as *resources* in a shared *context*. Components can build on res
 each other, making it possible to create components that offer highly sophisticated functionality
 with relatively little effort.
 
+Full documentation: https://asphalt.readthedocs.io/en/latest/
```
Comment (agronholm) [inline README.rst:37]:
```suggestion
Full documentation: https://asphalt.readthedocs.io/
```

**Comment by agronholm** [comment]
I guess there's no harm in adding the link. But I fear that for consistency I now have to update a dozen or so subprojects in a similar fashion.

**Pull request merged**